music_assistant/helpers/tags.py

```diff
@@ -64,7 +64,15 @@ def split_artists(
     # when not using the multi artist tag, the artist string may contain
     # multiple artists in freeform, even featuring artists may be included in this
     # string. Try to parse the featuring artists and separate them.
-    splitters = ("featuring", " feat. ", " feat ", "feat.")
+    splitters = (
+        " featuring ",
```

Comment:
do we need a title case version of all of them ?
so both " featuring " and "Featuring " - I can also simply do this programaticlaly to make the list easier

Reply:
Yeah maybe programmatically to save this list getting too long?

Reply:
I have removed the Ft. and added some more punctuation based joiners